test/unit/component/registry/css-registry.test.ts

```diff
@@ -49,6 +49,24 @@ describe('manage css classes for BPMN cells', () => {
     expect(cssRegistry.getClassNames(bpmnElementId)).toEqual(['class-name-1', 'class-name-2', 'class-name-3', 'class-name-4']);
   });
 
+  it('addClassNames should return true, when at least one new class name is not registered', () => {
+    const bpmnElementId = 'bpmn-id';
+    cssRegistry.addClassNames(bpmnElementId, ['class-name-1']);
+
+    const result = cssRegistry.addClassNames(bpmnElementId, ['class-name-1', 'class-name-2']);
+
+    expect(result).toBeTruthy();
+  });
+
+  it('addClassNames should return false, when all the new class names are registered', () => {
+    const bpmnElementId = 'bpmn-id';
+    cssRegistry.addClassNames(bpmnElementId, ['class-name-1', 'class-name-2']);
+
+    const result = cssRegistry.addClassNames(bpmnElementId, ['class-name-2', 'class-name-1']);
+
+    expect(result).toBeFalsy();
+  });
```

Comment:
❓ why not enriching the existing tests instead of adding new tests?

Reply:
I hesitated to do that. But the problem is that we test 2 things: the registration & the returned result.
 So the name of the test wiil be to long and, for me, it adds complexity on the test. A unit test should be simple.